tokens/tsm-report.go

```diff
@@ -0,0 +1,37 @@
+package tokens
+
+import (
+	"encoding/json"
+
+	"github.com/fxamacker/cbor/v2"
+)
+
+const TSMReportMediaType = "application/vnd.veraison.tsm-report+cbor"
```

Comment:
missing JSON media type

```suggestion
const (
	TSMReportMediaTypeCBOR = "application/vnd.veraison.tsm-report+cbor"
	TSMReportMediaTypeJSON = "application/vnd.veraison.tsm-report+json"
)
```